Packs/Hey/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Hey",
+    "name": "Hey (deprecated)",
     "description": "Use rakyll/hey to test a web application with a load of requests.",
```

Comment:
Add deprecated to the description as well

Reply:
added